keyserver/server.go

```diff
@@ -38,7 +42,8 @@ type ConiksServer struct {
 	stop     chan struct{}
 	waitStop sync.WaitGroup
 
-	secretKey crypto.SigningKey
+	db        kv.DB
+	secretKey sign.PrivateKey
```

Comment:
I think it could be removed for now. I just re-checking the source code, and `server.secretKey` is being used nowhere.